app/views/components/_autocomplete.html.erb

```diff
@@ -4,12 +4,15 @@
   label ||= nil
   value ||= nil
   options ||= []
+  local_assigns[:margin_bottom] ||= 6
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
```

Comment:
Could you link to this helper in the commit message? Also, what is the benefit of using the helper?

Reply:
Could you also add why "6" is being used as the default to the commit message as well?

Reply:
Commit message updated. The [shared helper](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md#shared-helper) is basically a way of not having to write the same bit of code for more than one component - it handles things like margins, heading levels, etc. in a consistent way, and already has tests written for it so we can keep testing per component for these things to a minimum.

Reply:
This ☝️  would be a great paragraph to add to the commit message!